### PR TITLE
Translation commands should not tell about the old app/ directory

### LINKED ISSUE #28816: Translation commands should not talk about the old "app/" directory since 3.4
**Symfony version(s) affected**: 3.4

**Description**  
`TranslationUpdateCommand` and `TranslationDebugCommand` mentions "the app/ folder" at several places (e.g. [here](https://github.com/symfony/symfony/blob/3.4/src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php#L83)) while the `translator.default_path` exists and defaults to `translations/` since 3.4.

**How to reproduce**  
Not really a bug, just a misleading/wrong mention.

**Possible Solution**  
Maybe talk about the configured `translator.default_path` and `twig.default_path` instead of `app/` (which implicitly means kernel.root_dir).

### PR DESCRIPTION
**Pull request by e-moe**
Fixes #28816 Translation commands should not talk about the old "app/" directory since 3.4

| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #28816 
| License       | MIT
| Doc PR        | n/a

### CODE REVIEW

**Review by yceruto — COMMENTED** [review]
Hi @e-moe, thinking a little bit more about this, I'm not sure that using `$this->defaultTransPath` is a good idea. Even if it's the default path for Sf4 apps, the `src/Resources/translations` still has priority and someone who uses it could face an inacurrate description again.

What about changing it to "default translations folder" everywhere? i.e. without being acurrate about the path.

PS: There is still a change to make in `$currentName = 'app folder'`.

**Comment by e-moe** [comment]
@chalasr , @yceruto , thanks for your comments. Messages updated to be less precise + "directory" instead of "folder"

File: src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php [hunk 1]
```diff
@@ -86,7 +86,7 @@ protected function configure()
             ->setHelp(<<<'EOF'
 The <info>%command.name%</info> command helps finding unused or missing translation
 messages and comparing them with the fallback ones by inspecting the
-templates and translation files of a given bundle or the app folder.
+templates and translation files of a given bundle or the default translation deirecoty.
```
Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php]: typo: directory

Comment (nicolas-grekas) [inline src/Symfony/Bundle/FrameworkBundle/Command/TranslationDebugCommand.php]: typo: directory
plural for "translations" (same below)

**Comment by e-moe** [comment]
@nicolas-grekas , thanks for review. fixed

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]
ok for master (most 3.4 projects are still on the "old" directory structure)

**Comment by fabpot** [comment]
Thank you @e-moe.

**Pull request merged**